### PR TITLE
WIP: Add validation

### PR DESCRIPTION
**Pull request by evilsquirr3l**

### CODE REVIEW

File: Business.Implementation/Validations/CategoryValidation.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using Business.Models;
+
+namespace Business.Implementation.Validations
+{
+    public static class CategoryValidation
+    {
+        public static void ValidateProduct(CategoryDto categoryDto)
```
Comment (evilsquirr3l) [inline Business.Implementation/Validations/CategoryValidation.cs:7]: ValidateCategory

File: Business.Implementation/Validations/CategoryValidation.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            if (string.IsNullOrEmpty(categoryDto.Name))
+            {
+                throw new ValidationException("Name can't be empty.");
+            }
+
+            if (string.IsNullOrEmpty(categoryDto.Description))
+            {
+                throw new ValidationException("Description can't be empty");
+            }
+
```
Comment (evilsquirr3l) [inline Business.Implementation/Validations/CategoryValidation.cs:19]: Delete blank line

File: Business.Implementation/Validations/CategoryValidation.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿using Business.Models;
+
+namespace Business.Implementation.Validations
+{
+    public static class CategoryValidation
+    {
+        public static void ValidateProduct(CategoryDto categoryDto)
+        {
+
```
Comment (evilsquirr3l) [inline Business.Implementation/Validations/CategoryValidation.cs:9]: Delete blank line

File: Business.Implementation/Validations/CartValidation.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    {
+        public static void ValidateProduct(CartDto cartDto)
+        {
+
+            if (cartDto.TotalPrice < 0)
+            {
+                throw new ValidationException("Price can't be of negative value");
+            }
+        }
+    }
```
Comment (evilsquirr3l) [inline Business.Implementation/Validations/CartValidation.cs:15]: Do we really need this cart validation?

Comment (Bybo4ka) [reply Business.Implementation/Validations/CartValidation.cs:15]: There was a history of unexpected bugs through development history, like when ghandi goes mad in civilization game series and starts to nuke everyone, so yes, I think it won't be abundant

Comment (evilsquirr3l) [reply Business.Implementation/Validations/CartValidation.cs]: You're fired.

**Pull request merged**